Userland/Libraries/LibChess/Chess.cpp

```diff
@@ -259,6 +259,16 @@ Board::Board()
     set_piece(Square("h8"), { Color::Black, Type::Rook });
 }
 
+Board Board::clone_cleared() const
```

Comment:
This sounds like the board is cleared, which isn't the case (and makes no sense). `clone_without_history` or `clone_current_state` maybe?

Reply:
Sure, I'm going with `clone_without_history`!